pages/accountLists/[accountListId]/contacts.page.tsx

```diff
@@ -22,7 +22,7 @@ const ContactsPage: React.FC = () => {
       </Head>
       <Box height="100vh" display="flex" overflow-y="scroll">
         <Box width="20vw">
-          <ContactFilters accountListId={accountListId as string} />
+          <ContactFilters accountListId={accountListId.toString()} />
```

Comment:
It's strange that this is the only place where ```accountListId``` isn't typed as a string. Not sure which method of typing this I prefer... what made you decide in favor ```.toString()``` if I may ask?

Reply:
Here, accountListId is coming from the router, and I guess the type of accountListId from that source is `Namespace<string>`.  Here's another page in the app that is fetching accountListId from the router, and is converted from `Namespace` to `string`: https://github.com/CruGlobal/mpdx-react/blob/main/pages/accountLists/%5BaccountListId%5D/tasks.page.tsx#L77

I really don't know what the best way to handle this is, I thought this might be a more natural way to convert it than forcefully casting it.

Reply:
VSCode says it's a `string | string[] | undefined` for me. It's coming from the `querystring` module.
```ts
interface ParsedUrlQuery extends NodeJS.Dict<string | string[]> { }
```
Seems like casting to a string makes sense since I don't see any way of telling the router what the types of its variables are. Maybe we should make a hook for this.